src/bootstrap/config.rs

```diff
@@ -849,6 +851,7 @@ impl Config {
             set(&mut config.codegen_tests, rust.codegen_tests);
             set(&mut config.rust_rpath, rust.rpath);
             set(&mut config.jemalloc, rust.jemalloc);
+            set(&mut config.mimalloc, rust.mimalloc);
```

Comment:
What happens if both are set? Will cargo give a compile error? If not, can you add a check in bootstrap?

Reply:
It will give the following compilation error, if both are enabled: <pre><code>error: Package `rustc-main v0.0.0 (/home/jkettune/rust/compiler/rustc)` does not have the feature `jemallocmimalloc`</code></pre>

It is not a that nice error, but maybe it will do for the transition period.

Reply:
I believe this is due to https://github.com/rust-lang/rust/pull/81782/files?file-filters%5B%5D=.example&file-filters%5B%5D=.rs&file-filters%5B%5D=.toml#diff-b46049dee2a9f04cac5ccc2e6820ecf7ef17d996c6f7258c85759cd48937b1f9R567-R572 which will concatenate both feature flags